osu.Game.Rulesets.Taiko/Difficulty/TaikoDifficultyCalculator.cs

```diff
@@ -129,14 +139,14 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
             {
                 StarRating = starRating,
                 Mods = mods,
-                RhythmDifficulty = rhythmRating,
+                RhythmDifficulty = rhythmRating * 8,
```

Comment:
That doesn't look intentional. If it is it needs an explanation

Reply:
oops... that was for a last minute test post-review